forward_linker/obs-linkr.py

```diff
@@ -29,10 +33,17 @@ def link_title(title, txt):
         txt_to_link = updated_txt[m.start() + offset:m.end() + offset]
         
         # where is the next ]]?
-        next_closing_index = updated_txt.find("]]", m.end() + offset)
+        next_closing_index = updated_txt.find("]", m.end() + offset)
         # where is the next [[?
-        next_opening_index = updated_txt.find("[[", m.end() + offset)   
-        
+        next_opening_index = updated_txt.find("[", m.end() + offset)   
+        # find if it is in a hyperlink
+        last_space = updated_txt.rfind(' ', 0, m.start()+offset)
+        last_newline = updated_txt.rfind('\n', 0, m.start()+offset)
+        last_sep = max(last_space, last_newline)
+        if last_sep == -1:
+            last_sep = 0
+        if len(list( re.finditer("https://|file://|ftp://|www\.", updated_txt[last_sep:m.start()+offset]) )):
+            continue
```

Comment:
I believe this is where I fixed the web links thing. Sorry, I am unable to provide a detailed response at this time because I got a little obsessed with this code so have made so many changes to suit my needs that I have forgotten many things about the original changes I made that I put up for pull request.

I totally understand your sentiment about not modifying the whole vault. That could cause irreversible damage.